src/base/bittorrent/statistics.cpp

```diff
@@ -84,7 +84,7 @@ void Statistics::gather()
 
 void Statistics::save() const
 {
-    const qint64 now = QDateTime::currentMSecsSinceEpoch();
+    const qint64 now = QDateTime::currentSecsSinceEpoch();
```

Comment:
It turns out that the save interval has now increased by 1000 times. Is this expected? Or am I mistaken?

Reply:
You are right, I missed it. Now fixed.
Also pushed another clean up commit.